test/test-cases/functional/saic/sai-api/test_sai_api_vnet_nhop_route.py

```diff
@@ -0,0 +1,52 @@
+from pprint import pprint
+
+import pytest
+
+# Constants
+SWITCH_ID = 5
+
+commands = [
+    {
+        "name": "route_entry_1",
+        "op": "create",
+        "type": "SAI_OBJECT_TYPE_ROUTE_ENTRY",
+        "key": {
+            "switch_id": "$SWITCH_ID",
+            "vr_id": "10",
+            "destination": "221.1.0.0/24"
+        },
+        "attributes": [
+            "SAI_ROUTE_ENTRY_ATTR_NEXT_HOP_ID",
+            "0"
```

Comment:
port ID will have to be updated to $PORT_0  and $PORT_1
as per https://github.com/opencomputeproject/SAI-Challenger/blob/main/common/sai.py line 490
self.create_alias(f"PORT_{idx}", 'SAI_OBJECT_TYPE_PORT', oid)